assets/wizards/popups/components/popup-action-card/index.js

```diff
@@ -74,12 +85,22 @@ const PopupActionCard = ( {
 			}
 		>
 			{ categoriesVisibility && (
-				<CategoryAutocomplete
-					value={ categories || [] }
-					onChange={ tokens => setCategoriesForPopup( id, tokens ) }
-					label={ __( 'Category filtering', 'newspack ' ) }
-					disabled={ sitewideDefault }
-				/>
+				<Fragment>
+					<CategoryAutocomplete
+						value={ campaignGroups || [] }
+						onChange={ tokens => setTermsForPopup( id, tokens, 'newspack_popups_taxonomy' ) }
```

Comment:
Nitpick, but this is set as a constant `Newspack_Popups::NEWSPACK_POPUPS_TAXONOMY` in that repo. Could we localize the taxonomy name using the constant so it's always in sync here?

Reply:
Addressed in https://github.com/Automattic/newspack-plugin/pull/767/commits/fb4cd9a522a99a9f45dbf67db43064bfe3e7a0ac and https://github.com/Automattic/newspack-popups/pull/363/commits/ac4441efb365f4a5684ba597319f6e4a8d9e3ff7